src/mediaclipmakarr/jobs.py

```
@@ -0,0 +1,762 @@
+            except asyncio.CancelledError:
+                await fail_job(
+                    self.engine,
+                    claimed.id,
+                    claimed.run_token,
+                    code="APP_SHUTDOWN",
+                    message="The application shut down before this job completed.",
+                    retryable=True,
+                )
+                await self.events.publish(claimed.id)
+                raise
+            except Exception as error:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)